ext/oauth-client/CRM/OAuth/Upgrader.php

```diff
@@ -36,4 +36,16 @@ public function upgrade_0001(): bool {
     return TRUE;
   }
 
+  /**
+   * Add support for tenancy
+   *
+   * @return bool
+   * @throws Exception
+   */
+  public function upgrade_5581(): bool {
+    $this->ctx->log->info('Applying oauth-client update 5581. Adding tenant column to the civicrm_oauth_client table.');
+    CRM_Upgrade_Incremental_Base::addColumn($this->ctx, 'civicrm_oauth_client', 'tenant', 'varchar(128) NULL COMMENT "Tenant ID" AFTER guid');
```

Comment:
@JKingsnorth I believe you can change this to be
```suggestion
     $this->addTask('Add Tenant ID to OauthClient', 'addColumn', 'civicrm_oauth_client', 'tenant', 'varchar(128) NULL COMMENT "Tenant ID" AFTER guid');
```

Reply:
Thanks @seamuslee001, that makes more sense. Done.